app/pulseapi/entries/views.py

```diff
@@ -196,20 +249,39 @@ class EntriesListView(ListCreateAPIView):
     )
     serializer_class = EntrySerializer
 
+    # which permissions allow this access to this model
+    permission_classes = [AllowAny]
+
     # Custom queryset handling: if the route was called as
     # /entries/?ids=1,2,3,4,... only return those entires.
     # Otherwise, return all entries (with pagination)
     def get_queryset(self):
+        user = self.request.user
+        is_superuser = user.is_superuser
+        is_moderator = user.has_perm('entries.change_entry')
+
+        # This initially used user.get_all_permissions, which
+        # works but is somewhat inefficient. Also note there is
+        # another "get all permissions" function that can lead
+        # to problems: https://stackoverflow.com/questions/2081061
+
+        queryset = False
+
+        # superusers and moderators get to see everything
+        if is_superuser is True or is_moderator is True:
+            queryset = Entry.objects.all()
```

Comment:
if that person is logged in as moderator or superuser, yes. Another option is to update the endpoint so that clients can pass an `&show_approved=1` to include those results, if they have viewing rights